examples/simple-game-server/main.go

```diff
@@ -787,8 +795,12 @@ func listContains(s *sdk.SDK, listName string, value string) (string, error) {
 	ok, err := s.Alpha().ListContains(listName, value)
 	if err != nil {
 		log.Printf("Error getting List %s contains value %s: %s", listName, value, err)
+		return strconv.FormatBool(ok), err
+	}	
+	if ok {
+		return "SUCCESS: " + strconv.FormatBool(ok) + "\n", nil
 	}
-	return strconv.FormatBool(ok), err
+	return "ERROR: " + strconv.FormatBool(ok) + "\n", err
```

Comment:
I realised what I think this should be:

```go
return "FOUND: " + strconv.FormatBool(ok) + "\n", nil
```

Which makes way more sense I think?

Also will need to adjust the unit test to match this change.

Reply:
Updated with FOUND in both the `main.go` and `gameserver_test.go` files.